client/src/partials/exchange/modal.js

```diff
@@ -27,10 +39,24 @@ function ExchangeModalController(Rates, Currencies, Session, $uibModalInstance,
 
   function submit(invalid) {
     if (invalid) { return; }
+    vm.data.enterprise_id = vm.enterpriseCurrencyId;
```

Comment:
@jniles 
That may be because we had to change the name of the column of table exchange_rate entreprise_currency_id  by enterprise_id, enterprise_id represents the currency of the enterprise in the exchange rate table